docs: schedule newsletter and email group enhancement (#83)
frappe/erpnext_documentation: erpnext_documentation/www/docs/user/manual/en/CRM/email_group.md

Click on Action > Import Subscribers and select a transaction like Lead. Click on Import and all the email IDs mentioned in the email IDs will be imported.

### 2.2 Email Templates

You can specify the confirmation email and welcome email template for the email group. A confirmation email will be sent to confirm a person's subscription to your newsletter. The welcome email is sent when the person confirms their subscription to your email newsletter.

> **a. Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription.<br />

Reviewer: ```suggestion
**Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription.
```

> **a. Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription.<br />

Reviewer: is this only for website or manual add? auto add via lead? add that information

> **Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription. By default, email generated via this template is sent on subscription via email group Website.

Reviewer: ```suggestion
**Confirmation Email Template:** This email template is used to replace the default template sent for newsletter subscription. By default, a confirmation email generated via this template is sent when someone subscribes to the Email Group Website.
```

> **b. Welcome Email Template:** This email template defines the email that is sent whenever a subscriber is added to the email group.

Reviewer: ```suggestion
**Welcome Email Template:** Whenever a subscriber is added to an email group, the template set here will be sent to them.
```
